samples/snippets/pom.xml

```diff
@@ -34,7 +34,7 @@
       <groupId>com.google.auth</groupId>
       <artifactId>google-auth-library-oauth2-http</artifactId>
       <version>1.5.3</version>
-    </dependency>
+    </dependency> 
```

Comment:
Is this change really necessary?

Reply:
nope... I am just testing to check if the presubmit samples work at 1.5.4 of auth.
I don't intent to merge this.